engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/HistoricActivityInstanceQueryDto.ftl

```diff
@@ -0,0 +1,18 @@
+<#include "/lib/commons/history-activity-instance.ftl" >
```

Comment:
is it possible to move it after `dto` and before `properties` macro

Reply:
👍